.github/workflows/cpu-inference.yml

```
@@ -2,13 +2,15 @@ name: cpu-inference
 
 on:
   pull_request:
-    paths-ignore:
-      - 'docs/**'
-      - 'blogs/**'
+    paths:
+      - '.github/workflows/cpu-inference.yml'
+      - 'requirements/**'
+      - 'deepspeed/inference/**'
```

```suggestion
      - 'deepspeed/inference/**'
      - 'deepspeed/__init__.py`
      - '!deepspeed/inference/v2/**' # exclude v2 dir
```

Let's also add the `__init__.py` file because it has `deepspeed.init_inference` and exclude any changes to `v2`